### DISCUSSION TITLE
New Event in the Scene Manager for Clients Done Loading Scene

### DISCUSSION THREAD

**Discussion by Katerlad** [FirstGearGames/FishNet#533] [Feature Request]
This was mentioned from ReignOfDwarfs developers.

Currently on on ClientPresenceChanged is captured as an event, which happens after observers built.

Some advanced users would like the ability to have ClientLoadedScenes as a seperate event. 

Looking at the code, ClientLoadedStartScenes is called at the proper location for this. 

It may be smart in the Args for the new ClientLoadedScenes to just have a bool that says it is the start scenes. so this event can capture any scene.

**Comment by FirstGearGames** [comment]
Not sure this is applicable anymore. On presence change has always had a before and after observer rebuild event though. Feel free to respond if I'm missing something.
